pkg/controller/migcluster/remotewatch.go

```diff
@@ -0,0 +1,102 @@
+/*
+Copyright 2019 Red Hat Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package migcluster
+
+import (
+	"os"
+
+	"github.com/fusor/mig-controller/pkg/controller/remotewatcher"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/event"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/source"
+
+	migrationv1alpha1 "github.com/fusor/mig-controller/pkg/apis/migration/v1alpha1"
+	velerov1 "github.com/heptio/velero/pkg/apis/velero/v1"
+)
+
+// RemoteManagerConfig specifies config options for setting up a RemoteWatch Manager
+type RemoteManagerConfig struct {
+	// rest.Config for remote cluster to watch Velero events on
+	RemoteRestConfig *rest.Config
```

Comment:
This could just be `RestConfig` within the context of the `RemoteManager` namespace.